tools/create_fat12.sh

```
@@ -0,0 +1,25 @@
+    echo "[FAT12] $IMG already exists, skipping (remove to rebuild)"
+    exit 0
+fi
+
+echo "[FAT12] Creating ${SIZE_KB}KB FAT12 image: $IMG"
+mkfs.fat -F 12 -C "$IMG" "$SIZE_KB"
+
+echo "[FAT12] Adding files from fat12_files/..."
+if [ -d fat12_files ]; then
+    for f in fat12_files/*; do
+        [ -f "$f" ] || continue
+        echo "  Adding: $(basename $f)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)